web-console/src/views/lookups-view.tsx

```diff
@@ -89,7 +89,7 @@ export class LookupsView extends React.Component<LookupsViewProps, LookupsViewSt
   componentDidMount(): void {
     this.lookupsGetQueryManager = new QueryManager({
       processQuery: async (query: string) => {
-        const tiersResp = await axios.get('/druid/coordinator/v1/tiers');
```

Comment:
Will this always return at least a `__default` tier? That is needed for the dropdown to work nicely.
If not it might make sense to add a UI side check for it...

Reply:
Good question. I think if no active servers to discover are available and no lookups are configured it could return an empty list. 

I've modified the UI side to check for this and at least provide the default lookup tier.. though I suppose there is no guarantee that it will have any available servers once they are discoverable, it still seems like better behavior.